WooCommerce/Classes/POS/Analytics/WooAnalyticsEvent+PointOfSale.swift

```diff
@@ -36,17 +35,56 @@ extension WooAnalyticsEvent {
                               properties: [Key.paymentsOnboardingState: onboardingState.reasonForAnalytics])
         }
 
-        static func addItemToCart(type: CartItemType, itemListType: ItemListType) -> WooAnalyticsEvent {
-            WooAnalyticsEvent(statName: .pointOfSaleAddItemToCart, properties: [
-                Key.itemType: type.analyticsValue,
-                Key.source: itemListType.addItemSourceAnalyticsValue
-            ])
+        static func addItemToCart(
+            source: WooAnalyticsEvent.PointOfSale.Source,
+            sourceType: WooAnalyticsEvent.PointOfSale.SourceType,
+            itemType: WooAnalyticsEvent.PointOfSale.ItemType,
+            productType: WooAnalyticsEvent.PointOfSale.ProductType? = nil,
+        ) -> WooAnalyticsEvent {
+            var properties: [String: String] = [
+                Key.source: source.rawValue,
+                Key.sourceType: sourceType.rawValue,
+                Key.itemType: itemType.rawValue
+            ]
+
+            if let productType = productType {
```

Comment:
super nit: (similar in L70)
```suggestion
            if let productType {
```